signac_dashboard/static/scss/app.scss

```diff
@@ -17,6 +17,26 @@ body {
   background-color: #2EB398;
 }
 
+.wrap-button-text {
```

Comment:
Do these CSS changes affect anything outside the Navigator module? We should probably use a CSS class for Navigator-specific CSS if it's meant to be confined to Navigator style.

Reply:
They don't conflict with any uses currently, but I think this is a good idea

Reply:
done